lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || this.analogPins;
+        this.pins = this.settings.pins || this.pins;
+        if (!this.pins.length) {
+          for (var i = 0; i < (this.settings.pinCount || 128); i++) {
```

Comment:
Ugh. Forget that I exist. I need to quit for the day. That's just sloppy brain right there.